baseframe/static/js/baseframe-bootstrap.js

```diff
@@ -292,6 +292,28 @@ window.Baseframe.Forms = {
         }
       }
     });
+  },
+  handleAjaxPost: function(config) {
+    $(config.formSelector).find('button[type="submit"]').click(function(event) {
```

Comment:
Expecting an id such as `#form` ties it to jQuery-specific syntax. `getElementById` does not need a `#`, so removing the jQuery dependency in future will become trouble-some if it requires a `#` to be prepended to the selector.

Reply:
It doesn't have to be `#form` it can be `.form` so even in the future if we replace this `document.querySelector` it should work